.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21]
```

Comment:
I've had to omit versions 6 and 7 here, which we are technically compatible with, but cannot enable due to development dependencies that are not compatible with these Node.js versions.